packages/smooth_app/lib/helpers/background_task_helper.dart

```diff
@@ -0,0 +1,233 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/CountryHelper.dart';
+import 'package:smooth_app/database/dao_product.dart';
+import 'package:smooth_app/database/local_database.dart';
+import 'package:smooth_app/query/product_query.dart';
+import 'package:task_manager/task_manager.dart';
+
+//Broadly classify the tasks into 2 categories
```

Comment:
Please be consistent with comments and use 3 dashes everywhere.
Also ensure to always have a space at beginning please

Reply:
`///` is for doc string, and `//` for standard comments, so I thought to use `///` only when necessary.

Reply:
Two dashes is within the code. Here, these are constants